src/wallet/rpcdump.cpp

```diff
@@ -88,6 +88,38 @@ bool GetWalletAddressesForKey(CWallet * const pwallet, const CKeyID &keyid, std:
     return fLabelFound;
 }
 
+bool ImportPrivateKey(CWallet * const pwallet, const UniValue& privkey, const std::string& label = "", const int64_t timestamp = 1)
+{
+    CKey key = DecodeSecret(privkey.get_str());
+    if (!key.IsValid()) throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Invalid private key encoding");
```

Comment:
In commit "[wallet] [rpc] Move private key import functionality out of importprivkey",

Nit, move throw to new line.

Reply:
Why?

Reply:
Consistency?

Reply:
Ahh, because of the other one below. Fixed.